package.json

```diff
@@ -29,6 +31,8 @@
   "dependencies": {
     "fp-ts": "^2.4.1",
     "locate-java-home": "^1.1.2",
+    "mkdirp": "^1.0.3",
+    "node-fetch": "^2.6.0",
```

Comment:
I've switched to using node-fetch instead of request, since the latter is [now deprecated](https://github.com/request/request/issues/3142)